docs/pages/export.rst

```diff
@@ -112,6 +112,24 @@ If you use the ``django_tables2.export.ExportMixin``, add an ``exclude_columns``
         exclude_columns = ("buttons", )
 
 
+Tablib Dataset Configuration
+----------------------------
+
+django-tables2 uses ``tablib`` to export the table data.
+You may pass kwargs to the ``tablib.Dataset`` via the TableExport constructor ``dataset_kwargs`` parameter::
+
+    exporter = TableExport("xlsx", table, dataset_kwargs={'title': 'My Custon Tab Title'})
```

Comment:
I'd use 'sheet name' instead of 'tab title'

Reply:
Done.